includes/admin/functions.php

```diff
@@ -20,6 +21,9 @@ function amp_init_customizer() {
 
 	// Add a link to the Customizer
 	add_action( 'admin_menu', 'amp_add_customizer_link' );
+
+	// Hook the functions to update the AMP cache on post update/deletion.
+	AMP_Cache_Utilities::amp_add_cache_update_actions();
```

Comment:
Good point.  The file is now `required` and the function hooked on `admin_init` at the bottom of amp.php.